projects/1.20-fabric/assets/neepmeat/neepmeat/lang/zh_cn.json

```
@@ -0,0 +1,1097 @@
+
+  "item.neepmeat.eldritch_enzymes_bucket": "怪异酶液桶",
+  "block.neepmeat.eldritch_enzymes": "怪异酶液",
+
+  "block.neepmeat.clean_ore_fat": "洁净%s脂肪",
+  "block.neepmeat.ore_fat": "污浊%s脂肪",
+  "ore_fat.neepmeat.iron_ingot.dirty": "污浊富铁脂肪",
+  "ore_fat.neepmeat.iron_ingot.clean": "洁净富铁脂肪",
+  "ore_fat.neepmeat.copper_ingot.dirty": "污浊富铜脂肪",
+  "ore_fat.neepmeat.copper_ingot.clean": "洁净富铜脂肪",
+  "ore_fat.neepmeat.gold_ingot.dirty": "污浊富金脂肪",
+  "ore_fat.neepmeat.gold_ingot.clean": "洁净富金脂肪",
+  "ore_fat.neepmeat.diamond.dirty": "污浊钻石脂肪",
+  "ore_fat.neepmeat.diamond.clean": "洁净钻石脂肪",
+  "ore_fat.neepmeat.lapis_lazuli.dirty": "污浊青金石脂肪",
+  "ore_fat.neepmeat.lapis_lazuli.clean": "洁净青金石脂肪",
+  "ore_fat.neepmeat.netherite.dirty": "污浊富下界合金脂肪",
+  "ore_fat.neepmeat.netherite.clean": "洁净富下界合金脂肪",
+
+  "block.neepmeat.milk": "奶",
+  "block.neepmeat.pasteurised_milk": "巴氏杀菌奶",
+  "item.neepmeat.pasteurised_milk_bucket": "巴氏杀菌奶桶",
+
+  "block.neepmeat.animal_feed": "动物饲料",
+  "item.neepmeat.animal_feed_bucket": "动物饲料桶",
+
+  "item.neepmeat.pinkdrink_bucket": "粉红饮料桶",
+  "block.neepmeat.pinkdrink": "粉红饮料",
+
+  "item.neepmeat.compressed_air_bucket": "压缩空气桶",
+  "block.neepmeat.compressed_air": "压缩空气",
+
+  "item.neepmeat.tank_minecart": "储罐矿车",
+  "item.neepmeat.small_compressor_minecart": "小型压缩机矿车",
+
+  "item.neepmeat.essential_saltes": "精盐：%s",
+  "item.neepmeat.essential_saltes.lore_0": "用于精卵合成机以复制生物体。",
+  "item.neepmeat.essential_saltes.meat": "所需肉浆液：%s",
+  "item.neepmeat.mob_egg": "生物卵：%s",
+  "item.neepmeat.mob_egg.lore_0": "必须浸泡在血液中才会孵化。",
+
+  "item.neepmeat.compound_injector": "化合物注射器",
+  "item.neepmeat.compound_injector_lore_0": "使用时回复4颗心。",
+  "item.neepmeat.compound_injector_lore_1": "鼠标拿起%s点击物品栏中的注射器以重新装填。",
+
+  "effect.neepmeat.ash_preparation": "焚灰预备",
+  "item.minecraft.potion.effect.ash_preparation": "焚灰预备液",
+  "item.minecraft.splash_potion.effect.ash_preparation": "喷溅型焚灰预备液",
+  "item.minecraft.lingering_potion.effect.ash_preparation": "滞留型焚灰预备液",
+
+  "item.neepmeat.goggles": "增强现实眼睛",
+  "item.neepmeat.goggles.lore_0": "揭示不可见之物",
+
+  "item.neepmeat.farming_scutter": "农作爬虫",
+  "item.neepmeat.farming_scutter.lore_0": "收获农作物并重新种植。必须放置在耕地上。收获到的物品会送入放置时高亮的位置。\n潜行点击以取走。",
+  "entity.neepmeat.farming_scutter": "农作爬虫",
+
+  "entity.neepmeat.glome": "高维球体",
+  "entity.neepmeat.hound": "猎犬",
+  "entity.neepmeat.bovine_horror": "怖牛",
+
+  "message.neepmeat.implant.title": "植入物",
+  "message.neepmeat.item_implant.title": "物品植入物",
+  "message.neepmeat.implant.install": "植入物已嵌装：%s",
+
+  "item.neepmeat.pineal_eye": "松果眼",
+  "implant.neepmeat.pineal_eye": "松果眼",
+  "implant.neepmeat.pineal_eye.desc_0": "永久性揭示不可视之物。",
+
+  "item.neepmeat.extra_mouth": "额外口器",
+  "implant.neepmeat.extra_mouth": "额外口器",
+  "implant.neepmeat.extra_mouth.desc_0": "自动食用物品栏中的食物。会尽量不浪费，但也会在损失不少于两颗心时优先食用高饱和度的食物。",
+  "item.neepmeat.extra_knees": "强化膝盖",
+  "implant.neepmeat.extra_knee": "强化膝盖",
+  "implant.neepmeat.extra_knees.desc_0": "将步伐高度增至一格高。",

```

```suggestion
  "implant.neepmeat.extra_knees.desc_0": "将行走高度增至一格高。",
```
https://zh.minecraft.wiki/w/%E5%B1%9E%E6%80%A7